fedbadges/rules.py

```diff
@@ -222,8 +191,8 @@ def _get_candidates(self, msg: Message, tahrir: TahrirDatabase):
         # Remove None
         candidates = frozenset([e for e in candidates if e is not None])
 
-        # Exclude banned usernames
-        candidates = candidates.difference(self.banned_usernames)
+        # Exclude skipped usernames
+        candidates = candidates.difference(self.skip_users)
```

Comment:
Here, `self.skip_users` is the filename `"config.toml"`. We want it to be the actual list of usernames.

You can get that list by using the `self.config.get("skip_users", [])`.